instaclustr/resource_kafka_schema_registry_user.go

```diff
@@ -0,0 +1,76 @@
+package instaclustr
```

Comment:
As per the API - would it make sense to just create a generic resource and data source instead? Example - `instaclustr_bundle_user` ? That way we don't have to create all these separate resources that do the same thing.

Reply:
Working on it at the moment

Reply:
Implementation is done.